roles/setup-site/tasks/main.yml

```diff
@@ -65,13 +65,13 @@
     minutes: 1
 
 - name: Check drush can bootstrap site
-  shell: "{{ drush_alias }} status"
+  shell: "{{ drush_alias }} status --fields='Drupal bootstrap' --field-labels=0 --strict=0"
```

Comment:
I'm not s big fan of this check. Perhaps swapping out this for the API is best. The API might be slower than this check but perhaps we try it out.

Reply:
I can dig it.